curated_transformers/models/feedforward.py

```diff
@@ -31,12 +31,17 @@ def __init__(
         hidden_act: str = "gelu",
         hidden_width: int = 768,
         intermediate_width: int = 3072,
+        use_bias: bool,
```

Comment:
We don't want a default here?

Reply:
I think we should do the opposite here and remove all the defaults (like attention). I'll add a commit to do this.

Reply:
Removed all the defaults.